src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import * as S from "./styles";
+
+const profileContent: React.FC = () => {
+    const [name, setName] = useState<string | null>(null);
+    const [subject, setSubject] = useState<string | null>(null);
+    const [content, setContent] = useState<string | null>(null);
+  
+    const handleSubmit = (event: React.MouseEvent) => {
+      event.preventDefault();
+  
+      if (name === null) {
+        setName("");
+      }
+  
+      if (subject === null) {
+        setSubject("");
+      }
+  
+      if (content === null) {
+        setContent("");
+      }
+  
+      if (!name || !subject || !content) {
+        return;
+      }
+  
+      const body =
+        `Dear Winston,\n\n` +
+        `${content}\n\n` +
+        `Thank you.\n\n` +
+        `Best regards,\n\n` +
+        `${name}\n`;
+  
+      location.href =
```

Comment:
Try to explain what does `location.href` mean and `encodeURIComponent` mean

Reply:
location.href mean the url of the page and encodeURIComponent is to replace the values in the string , turning it into URL

Reply:
yup! actually escapes the character too